tools/pd-ctl/pdctl/command/keyspace_group_command.go

```diff
@@ -50,6 +51,15 @@ func newSplitKeyspaceGroupCommand() *cobra.Command {
 	return r
 }
 
+func newSplitRangeKeyspaceGroupCommand() *cobra.Command {
```

Comment:
Do we need to add a test for this?

Reply:
will add in separate pr. Already tested in dev.